pkg/tfbridge/provider.go

```diff
@@ -1340,12 +1340,28 @@ func generateResourceName(packageName tokens.Package, moduleName string, moduleM
 	return fmt.Sprintf("%s.%s.%s", packageName, moduleName, moduleMemberName)
 }
 
-// SetAutonaming will loop all resources with a name property, and will add an auto-name property.  It will skip
-// those that already have a name mapping entry, since those may have custom overrides set in the resource
-// declaration (e.g., for length).
+// SetAutonaming applies [AutoName] to every required or optional resource property that is literally called "name".
```

Comment:
Do we have good docs elsewhere? I found the existing docs pretty laconic, itching to expand a bit.